test/functional/mempool_sigoplimit.py

```diff
@@ -133,6 +136,49 @@ def test_sigops_limit(self, bytes_per_sigop, num_sigops):
         assert_equal(entry_parent['descendantcount'], 2)
         assert_equal(entry_parent['descendantsize'], parent_tx.get_vsize() + sigop_equivalent_vsize)
 
+    def test_sigops_package(self):
+        # Make a 2-transaction package which fails vbyte checks even though
+        # separately they would work.
+        self.restart_node(0, extra_args=["-bytespersigop=5000"] + self.extra_args[0])
+
+        _, pubkey = generate_keypair()
+        amount_for_bare = 50000
+        tx_parent_dict = self.wallet.create_self_transfer(fee=Decimal("3"))
+        tx_parent_utxo = tx_parent_dict["new_utxo"]
+        tx_parent = tx_parent_dict["tx"]
+        tx_parent.vout.append(CTxOut(amount_for_bare, keys_to_multisig_script([pubkey], k=1)))
+        tx_parent.vout[0].nValue -= amount_for_bare
+        tx_parent.calc_sha256()
+        tx_parent_utxo["txid"] = tx_parent.hash
```

Comment:
nit 5c48501e9f1a3da5307b1bee08396598398311e8: you can also just do this
```suggestion
        tx_parent_utxo["txid"] = tx_parent.rehash()
```

Reply:
done